src/tests/utils/parse_args_test.go

```diff
@@ -0,0 +1,74 @@
+package tests
```

Comment:
once you unexport parseArgs you'll need to make this the same as subcommand's package to call the package-local parseArgs function

Reply:
done